File: python practice/20260611_Hello_World_PC.py
import sys

def solve():
    input_data = sys.stdin.read().split()
    if not input_data:
        return
    
    T = int(input_data[0])
    
    output_blocks = []
    
    for i in range(1, T + 1):
        n = int(input_data[i])
        
        block = "Hello World\n" * n
        output_blocks.append(block)
    
    sys.stdout.write("\n".join(output_blocks))

if __name__ == "__main__":
    solve()

File: C Language/20260611_Hello_World_PC.c
#include <stdio.h>

int main() {
    int T, n;

    if (scanf("%d", &T) != 1) {
        return 0;
    }

    for (int i = 0; i < T; i++) {
        if (scanf("%d", &n) == 1) {
            // 印出 n 次 Hello World
            for (int j = 0; j < n; j++) {
                printf("Hello World\n");
            }
            
            if (i < T - 1) {
                printf("\n");
            }
        }
    }

    return 0;
}